### PR TITLE
ProcTankSpaceplane: added tooling

### PR DESCRIPTION
**Pull request by Schnobs**
Problem: 
Shielded Proctank was not part of the general RP-1 overhaul and still uses the old system of unlocking sizes by techlevel (no tooling). Also has 100% utilization.

Solution:
Retrofitted tooling and applied a max utilization.

### CODE REVIEW

File: GameData/RP-0/Parts/RFProcTanks.cfg [hunk 1]
```diff
@@ -423,8 +423,40 @@
...
+	}
+
+}
+
 @PART[proceduralTankRealFuels]:FOR[zzzRP-0]
 {
 	%category = -1
 	%techRequired = ORPHANS
 }
+
```
Comment (Tidal-Stream) [inline GameData/RP-0/Parts/RFProcTanks.cfg]: Why is this line added? No need for it.

Comment (Schnobs) [reply GameData/RP-0/Parts/RFProcTanks.cfg]: I honestly cannot remember anymore. I guess it's either carried over from whatever current config I copied, or a remnant of the old config that's escaped deletion.

Comment (Tidal-Stream) [reply GameData/RP-0/Parts/RFProcTanks.cfg]: It's OK, it was merged already.
